### DISCUSSION TITLE
End User License Agreement

### DISCUSSION THREAD

**Discussion by clementbiron** [OpenTermsArchive/terms-types#27] [Submitted]
To find out if we would need `End User License Agreement` document type, let's list in this discussion the documents that would be defined by this type.

A proposal for a short definition: an end user license agreement (EULA) sets the legal terms between a software publisher and the user, outlining rights and restrictions for software use. It typically includes information about the software's features, limitations, restrictions, and the publisher's warranty and liability terms.

| URL                                                          | Title               | Service              | Service acts as     | Jurisdiction |
|--------------------------------------------------------------|---------------------|----------------------|---------------------|--------------|
| [URL](https://legal.jasper.ai/#eula) | End User License Agreement | Jasper             | AI provider |           |
| [URL](https://www.apple.com/legal/internet-services/itunes/dev/stdeula/)  | Licensed application end user license agreement                     | Apple iTunes | Media service |              |
| [URL]()                                                      |                     | _None yet, add one!_ |                     |              |

**Comment by MattiSG** [comment]
What is the difference between the EULA and the Terms of Use? 🤔

**Reply by clementbiron** [reply]
An EULA is the agreement for someone who downloads a software. Terms of Use is wider reaching, and regulates the entire agreement between both parties, from the moment a visitor lands on the homepage, to the moment they download any software.
